Realm/ObjectServerTests/RLMSyncTestCase.mm

```diff
@@ -675,21 +469,21 @@ - (void)resetSyncManager {
         XCTestExpectation *ex = [self expectationWithDescription:@"Wait for logout"];
         [exs addObject:ex];
         [user logOutWithCompletion:^(NSError *) {
+            // Sessions are removed from the user asynchronously after a logout.
```

Comment:
I don't think moving this code here works. It's adding expectations to the array after the call for waitForExpectations and then depending on that call waiting on the new ones, and if that does actually work it's only by coincidence.